File: JAVA/PROGRAM/Mahasiswa.java
class Mahasiswa{
    private String nama;
    private String NIM;
    private String prodi;
    private String fakultas;
    Mahasiswa()// CONSTRUCTOR
    {
    }
    void setNama(String a) // PROSEDUR UNTUK SET NAMA
    {
        this.nama = a;
    }
    void setNIM(String b) // PROSEDUR UNTUK SET NIM
    {
        this.NIM = b;
    }
    void setProdi(String c) // PROSEDUR UNTUK SET PRODI
    {
        this.prodi = c;
    }
    void setFakultas(String d)// PROSEDUR UNTUK SET FAKULTAS
    {
        this.fakultas = d;
    }
    String getNama()//PROSEDUR UNTUK MENDAPATKAN NAMA
    {

        return this.nama;
    }
    String getNIM()//PROSEDUR UNTUK MENDAPATKAN NIM
    {
        return this.NIM;
    }
    String getProdi()//PROSEDUR UNTUK MENDAPATKAN PRODI
    {

        return this.prodi;
    }
    String getFakultas()//PROSEDUR UNTUK MENDAPATKAN FAKULTAS
    {
        return this.fakultas;
    }
    
}

File: c++/program/Mahasiswa.cpp
#include <iostream>
#include <string>
using namespace std;
class Mahasiswa
{
private: // private atribut
    string nama;
    string NIM;
    string prodi;
    string fakultas;

public:
    Mahasiswa() // CONSTRUCTOR
    {
    }
    void setNama(string a) // SET NAMA
    {
        nama = a;
    }
    void setNIM(string b) // SET NIM
    {
        NIM = b;
    }
    void setProdi(string c) // SET PRODI
    {
        prodi = c;
    }
    void setFakultas(string d) // SET FAKULTAS
    {
        fakultas = d;
    }
    string getNama() // GET NAMA
    {

        return nama;
    }
    string getNIM() // GET NIM
    {
        return NIM;
    }
    string getProdi() // GET PRODI
    {

        return prodi;
    }
    string getFakultas() // GET FAKULTAS
    {
        return fakultas;
    }
    ~Mahasiswa() // DESTRUCTOR
    {
    }
};